### DISCUSSION TITLE
Hold event parameter?

### DISCUSSION THREAD

**Discussion by JoergM-D** [nguyenquyhy/Flight-Tracker-StreamDeck#254] [Q&A]
Hi, 
thanks for this best plugin ever!
I love and try to to build a profile for the C172 classic. Here I want to  use  for example "VOR1_OBI_DEC", which shoud turn the knob as long as I hold the button. All my tries didn't work so far.  Does somebody could help me? 

Cheers Jörg

**Comment by JoergM-D** [comment]
Solved

**Reply by nguyenquyhy** [reply]
How? 😀

**Comment by JoergM-D** [comment]
Hi Nguyen,
using  the Preset Increase/Decrease Button and then the function VOR1, VOR2 and ADF

Greetings from EDDH
Jörg
